### PR TITLE
Feature/provider onboarding

### PR DESCRIPTION
**Pull request by lemredd**
Onboarding for provider role users.

### CODE REVIEW

File: next.config.mjs [hunk 1]
```diff
@@ -33,6 +33,19 @@ const nextConfig = {
       },
     ],
   },
+
```
Comment (lemredd) [inline next.config.mjs:36]: take note

File: utils/supabase/middleware.ts [hunk 2]
```diff
@@ -59,15 +59,15 @@ export async function updateSession(request: NextRequest) {
...
+      case 'SKR':
         if (!request.nextUrl.pathname.startsWith('/skr')) {
           return Redirect('/skr')
         }
         if (
           !profile?.is_completed &&
           !request.nextUrl.pathname.startsWith('/skr/setup')
         ) {
-          return Redirect('/skr/setup/skills')
+          return Redirect('/skr/setup')
```
Comment (lemredd) [inline utils/supabase/middleware.ts:70]: take note

File: next.config.mjs [hunk 3]
```diff
@@ -33,6 +33,19 @@ const nextConfig = {
...
+      source: '/skr/setup',
+      destination: '/skr/setup/profile',
+      permanent: true,
+    },
+    {
+      source: '/pdr/setup',
+      destination: '/pdr/setup/project',
+      permanent: true,
+    }
+  ]
```
Comment (lemredd) [inline next.config.mjs:48]: take note

File: utils/supabase/middleware.ts (hunk 2 above)
Comment (rnldvrgl) [reply utils/supabase/middleware.ts:70]: this is noted

File: next.config.mjs (hunk 3 above)
Comment (rnldvrgl) [reply next.config.mjs:48]: noted again

**Pull request merged**